crates/sui-indexer-alt/src/benchmark.rs

```diff
@@ -14,21 +14,14 @@ pub struct BenchmarkArgs {
     /// Path to the local ingestion directory to read checkpoints data from.
     #[arg(long)]
     ingestion_path: PathBuf,
-
-    /// Only run the following pipelines. If not provided, all pipelines will be run.
-    #[arg(long, action = clap::ArgAction::Append)]
-    pipeline: Vec<String>,
```

Comment:
Back to my comment on the previous config, i wonder if keeping this as an Arg has value for quickly iterating vs needing to edit a file. That is, have the CLI arg be an override for what is specified in the file.

Reply:
Yeah we could definitely add that -- I guess with config merging, the thinking was that each pipeline was in its own config, and so you could merge together the configs for pipelines you wanted to use and then run that, but that's still a second step, and I don't see a harm in adding multiple ways of manipulating the configs.